libs/core/src/components/pds-radio/pds-radio.scss

```diff
@@ -1,22 +1,26 @@
 :host {
-  --background-color: var(--pine-color-base-white);
-  --background-color-hover: var(--pine-color-neutral-grey-100);
-  --background-color-disabled: var(--pine-color-neutral-grey-200);
-  --border: var(--pine-border-interactive);
-  --border-color-hover: var(--pine-border-interactive-hover-color);
-  --border-color-disabled: var(--pine-color-neutral-grey-300);
+  --border-interactive: var(--pine-border-interactive-default);
   --border-radius: 50%;
   --box-shadow-focus: 0 0 0 2px var(--color-focus);
   --box-shadow-focus-error: 0 0 0 2px var(--color-invalid-focus);
-  --color: var(--pine-color-neutral-charcoal-200);
-  --color-checked: var(--pine-color-neutral-charcoal-500);
-  --color-disabled: var(--pine-color-neutral-grey-500);
-  --color-focus: var(--pine-color-primary-200);
+
+  --color: var(--pine-color-charcoal-200);
```

Comment:
Would this be `color-text-default`?

Reply:
@anechol This one is technically not being used anywhere.